data.sql

```diff
@@ -1,6 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '02/03/20', 0, true,10.23);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '15/11/18',2, true,8);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '01/07/21', 1, false,15.04);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Davimon', '05/12/17', 5, true,11);
\ No newline at end of file
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '02/03/20', 0,true,10.23);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '11/15/18', 2,true,8);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '01/7/21', 1,false,15.04);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Davimon', '05/12/17', 5,true,11);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Chamandar', '02/08/20', 0, false, 11);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (6, 'Plantmon', '11/15/21', 2, true, 5.7);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (7, 'Squirtle', '04/02/93', 3, false, 12.13);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (8, 'Angemon', '06/12/05', 1, true, 45);
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235653153-17d6df3a-9e69-4fd2-acea-1c9d80d929b2.png)

- [ ] It would be nice if you insert the data manually and correct the weights of the animals in your data.sql file 👍 as suggested by the previous reviewer because right now what you have done is below ⬇️

1. First you have made all the weights negative which is not correct at all 😨
2. Secondly, you didn't push your code 🤔

Kindly make the changes in your data.sql file manually and then push your code 😊

Reply:
Manually inserted correct weight in the data.sql file and pushed. Thank you @HAMMAS-SALEEM